elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/metadata/models/Metric.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2019, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.datastores.aggregation.metadata.models;
+
+import com.yahoo.elide.annotation.Include;
+import com.yahoo.elide.datastores.aggregation.AggregationDictionary;
+import com.yahoo.elide.datastores.aggregation.metadata.enums.Format;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+import javax.persistence.Entity;
+import javax.persistence.ManyToOne;
+
+/**
+ * Special column for AnalyticView which supports aggregation.
+ */
+@EqualsAndHashCode(callSuper = true)
+@Include(rootLevel = true, type = "metric")
```

Comment:
In general, I wonder if metrics and dimensions should be root.  I'm wondering if the only objects that shoudl be root are tables, analyticViews, and the basic types.

Reply:
I would not change that for now. We can always determine this later I think.

Reply:
Remove root please.